contentful_cli/management.py

```diff
@@ -82,36 +82,57 @@ def echo_output(obj, echo_to_stdout, log_file):
+    Endpoint('unpublish-asset',         'delete', 'api',    '/spaces/{space_id}/assets/{asset_id}/published',                      'EV'),
+    Endpoint('archive-asset',           'put',    'api',    '/spaces/{space_id}/assets/{asset_id}/archived',                       'EV'),
+    Endpoint('unarchive-asset',         'delete', 'api',    '/spaces/{space_id}/assets/{asset_id}/archived',                       'EV'),
+    # get-locales is mis-named. It should be list-locales. We should remove it in a future version.
+    Endpoint('get-locales',             'get',    'api',    '/spaces/{space_id}/locales',                                          'E'),
+    Endpoint('list-locales',            'get',    'api',    '/spaces/{space_id}/locales/',                                         'Ec'),
+    Endpoint('post-locale',             'post',   'api',    '/spaces/{space_id}/locales/',                                         'ED'),
+    Endpoint('put-locale',              'put',    'api',    '/spaces/{space_id}/locales/{locale_id}',                              'ED'),
+    Endpoint('get-locale',              'get',    'api',    '/spaces/{space_id}/locales/{locale_id}',                              'E'),
+    Endpoint('delete-locale',           'delete', 'api',    '/spaces/{space_id}/locales/{locale_id}',                              'E'),
+    Endpoint('list-space-memberships',  'get',    'api',    '/spaces/{space_id}/space_memberships/',                               'Ec'),
+    Endpoint('post-space-membership',   'post',   'api',    '/spaces/{space_id}/space_memberships/',                               'ED'),
+    Endpoint('put-space-membership',    'put',    'api',    '/spaces/{space_id}/space_memberships/{membership_id}',                'ED'),
+    Endpoint('get-space-membership',    'get',    'api',    '/spaces/{space_id}/space_memberships/{membership_id}',                'E'),
+    Endpoint('delete-space-membership', 'delete', 'api',    '/spaces/{space_id}/space_memberships/{membership_id}',                'E'),
+    Endpoint('list-roles',              'get',    'api',    '/spaces/{space_id}/roles/',                                           'Ec'),
+    Endpoint('post-role',               'post',   'api',    '/spaces/{space_id}/roles/',                                           'ED'),
+    Endpoint('put-role',                'put',    'api',    '/spaces/{space_id}/roles/{role_id}',                                  'ED'),
+    Endpoint('get-role',                'get',    'api',    '/spaces/{space_id}/roles/{role_id}',                                  'E'),
+    Endpoint('delete-role',             'delete', 'api',    '/spaces/{space_id}/roles/{role_id}',                                  'E'),
+    Endpoint('get-environment',         'get',    'api',    '/spaces/{space_id}/environments/{environment_id}',                    ''),
```

Comment:
These are environment aware endpoints but don't use the same mechanism to specify the environment as other endpoints.